src/components/common/field-token.js

```diff
@@ -20,28 +20,78 @@
 
 import React from 'react';
 import styled from 'styled-components';
+import {ALL_FIELD_TYPES} from 'constants/default-settings';
 
-import {FILED_TYPE_DISPLAY, FIELD_COLORS} from '../../constants/default-settings';
+const FieldTokenFactory = () => {
+  const FieldTag = styled.div`
+    background-color: rgba(${props => props.color}, 0.2);
+    border-radius: 2px;
+    border: 1px solid rgb(${props => props.color});
+    color: rgb(${props => props.color});
+    display: inline-block;
+    font-size: 10px;
+    font-weight: 400;
+    padding: 0 5px;
+    text-align: center;
+    width: 40px;
+  `;
 
-const FieldTag = styled.div`
-  background-color: rgba(${props => props.color}, 0.2);
-  border-radius: 2px;
-  border: 1px solid rgb(${props => props.color});
-  color: rgb(${props => props.color});
-  display: inline-block;
-  font-size: 10px;
-  font-weight: 400;
-  padding: 0 5px;
-  text-align: center;
-  width: 40px;
-`;
+  const ORANGE = '248, 194, 28';
```

Comment:
why do you move these constant out from `default-settings` and into `field-token.js` ?

Reply:
these contants are being using just on the field-token, for all the other colors we're using the theme. that's why it makes more sense to move them inside the token component

Reply:
no, please leave the constatns in the constants folder, because they might be used by other files in the future

Reply:
Done